### DISCUSSION TITLE
A Newbie here, just want to know !

### DISCUSSION THREAD

**Discussion by Jikkesh** [Ismaestro/angular4-example-app#150] [General]
Hey Ismaestro,

Can I know what is the reason you have give standalone for components, I guess it looks more good with, adding some imports to app.module and easy to read. Any Specific reasons for following this design pattern ?

**Comment by Ismaestro** [comment]
Hi @Jikkesh !!
Nice question! I was wondering about that too, so I figured I'd ask the community about the pros and cons of sticking solely to standalone components. [Check it out!](https://stackoverflow.com/questions/74558393/best-practices-standalone-components-vs-modules-in-angular-14)

**Reply by Jikkesh** [reply]
I guess it may work good with Lazy Loading, Thank you letting me know about this concept and I can maybe create an another Branch of this with a Traditional Design of having Each Components and their Services together and Learn. Thank you

**Comment by mdk2025** [comment]
Hi @Ismaestro 

I am new to angular. how can a generate a component inside the following path with importing it in NgModule? or should i just use 
`ng generate component common --skip-import`

<img width="381" alt="image" src="https://github.com/Ismaestro/angular-example-app/assets/42651282/38867780-7d9f-447e-8484-cff5068530ef">
